### ISSUE TITLE
host network leaves container with no network

### ISSUE THREAD

**Issue by phynias** [nebula-orchestrator/worker#35]
## Expected/Wanted Behavior
container on host network

## Actual Behavior
no network set

## Steps to Reproduce the Problem
used the following config

```
        {
          "starting_ports": [],
          "containers_per": {"server": 1},
          "env_vars": {"ENV": "dev"},
          "docker_image" : "mine/sensu-client",
          "running": true,
          "networks": ["host"],
          "privileged": true,
          "devices": [],
          "volumes": []
        }
```
but inside the container ifconfig just showed lo nic.

**Comment by phynias** [comment]
yeah, looking in the docker-py api docs it says you just have to set network_mode=host
 ¯\_(ツ)_/¯

**Comment by naorlivne** [comment]
Yep that really all it was, fixed it in 2.0.0a (which is the alpha feature branch for 2.0.0) so it will be added to 2.0.0 & master once 2.0.0 is ready to be merged and becomes the latest version.
